src/driver/odoc_driver.ml

```diff
@@ -205,6 +211,7 @@ let run mode
     else []
   in
   Logs.debug (fun m -> m "XXXX Remaps length: %d" (List.length remaps));
+  Logs.app (fun m -> m "Starting the compilation process... \n%!");
```

Comment:
This newline and flush are probably unwanted. Logs does that again.

Reply:
Thanks!